index.html

```diff
@@ -0,0 +1,183 @@
+                    </div>
+                </div>
+                <div class="program-card">
+                    <div class="card-icon">
+                        <img src="images/icons/icon4.png" alt="icon 4">
+                        <h3>Heirs</h3>
+                    </div>
+                    <div class="card-description">
+                        <p>And if children, then heirs; heirs of God, and joint-heirs with Christ; if so be that we suffer with him, that we may be also glorified together. <br> - Romans 8:17</p>
+                    </div>
+                </div>
+                <div class="program-card">
+                    <div class="card-icon">
+                        <img src="images/icons/icon5.png" alt="icon 5">
+                        <h3>Confess</h3>
+                    </div>
+                    <div class="card-description">
+                        <p>Whosoever therefore shall confess me before men, him will I confess also before my Father which is in heaven. <br> - Matthew 10:32</p>
+                    </div>
+                </div>
+            </div>
+            <div class="program-btn">
+                <button type="button">Join us and Spread the Love of Jesus</button>
+            </div>
+        </section>
+
+        <section id="testimonials">
+            <div class="testimonial-title">
+                <h2>Testimonials</h2>
+                <hr>
+            </div>
+            <div class="testimonial-container">
+            </div>
+            <div class="testimonial-btn">
+                <button type="button">More <img src="images/icons/down-arrow.svg" alt="down icon"></button>
+            </div>
+        </section>
+
+        <section id="partners">
+            <div class="partner-title">
+                <h2>Partner</h2>
+                <hr>
+            </div>
+
+            <div class="partner-card">
+                <div class="partner-logo">
+                    <img src="images/partner1.png" alt="partner 1">
+                </div>
+                <div class="partner-logo">
+                    <img src="images/partner2.png" alt="partner 2">
+                </div>
+                <div class="partner-logo">
+                    <img src="images/partner3.png" alt="partner 3">
+                </div>
+                <div class="partner-logo">
+                    <img src="images/partner4.png" alt="partner 4">
+                </div>
+                <div class="partner-logo">
+                    <img src="images/partner5.png" alt="partner 5">
+                </div>
+            </div>
+        </section>
+    </main>
+    <footer id="footer">
+        <div class="footer-logo">
+            <img src="images/logo.jpg" alt="logo">
+        </div>
+        <div class="footer-content">
+            <p>2022 Kefi Unearthed. All rights Reserved</p>
+        </div>
+    </footer>
```

Comment:
I added space to my project to see how it would turn out. I didn't like it. I would like to please keep my design customized to my preference. 🙏 Please and thank you.